_includes/style/icon.html

```diff
@@ -5,5 +5,6 @@
 Parameters:
 
 type: A name that matches an alert. Valid values are "success", info", "warning" and "danger".
+class: A string to be used for additional "class" values
```

Comment:
Could add `(optional)` to this parameter to make it clear that it is not required.